src/domexpression.js

```diff
@@ -31,6 +34,21 @@ var _ = Mavo.DOMExpression = $.Class({
 			this.expression = expression;
 		}
 
+		if (this.attribute?.startsWith("mv-attr-")) {
+			this.originalAttribute = this.attribute;
+			this.attribute = this.attribute.replace("mv-attr-", "");
+
+			if (["http://www.w3.org/2000/svg", "http://www.w3.org/1998/Math/MathML"].includes(this.element.namespaceURI)) {
+				this.attribute = Mavo.getProperAttributeCase(this.element, this.attribute);
+			}
+
+			this.fallback = this.fallback || Mavo.Primitive.getValue(this.element, {attribute: this.attribute});
```

Comment:
```suggestion
			this.fallback ||= Mavo.Primitive.getValue(this.element, {attribute: this.attribute});
```

(also do we need `||` or `??` here?)

Reply:
I assume that some falsy values (e.g., `0`) might be perfectly valid fallbacks, and we don't want to replace them with any other value. So, I'm leaning towards `??=`.